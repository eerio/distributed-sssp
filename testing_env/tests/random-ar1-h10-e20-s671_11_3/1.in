11 4 7
4 3 14
4 5 12
4 0 17
4 7 16
5 4 12
5 6 10
5 0 17
5 7 5
6 5 10
6 7 0
7 6 0
7 8 14
7 10 8
7 9 15
7 4 16
7 5 5
