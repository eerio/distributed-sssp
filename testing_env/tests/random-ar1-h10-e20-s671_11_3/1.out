17
17
12
12
